src/compute-client/src/controller/instance.rs

```diff
@@ -1455,6 +1455,9 @@ where
             ComputeResponse::PeekResponse(uuid, peek_response, otel_ctx) => {
                 self.handle_peek_response(uuid, peek_response, otel_ctx, replica_id)
             }
+            ComputeResponse::CopyToResponse(id, response) => {
+                Some(ComputeControllerResponse::CopyToResponse(id, response))
+            }
```

Comment:
@teskje Does this look fine to you? Removed the code which was targeting the replica, so it's going to send back responses across all of them.

Reply:
Ah, you still need to make sure that you only return the first replica's response! What the replica-targeting does is that it only returns the response from a selected replica and ignore all others. We don't want that but the untargeted behavior which is returning the response from the fastest replica and ignoring all others.

So we still want the `copy_tos` field, but it should be a `BTreeSet<GlobalId>` (the values in the `BTreeMap` were the targeted replicas).

Reply:
Ah that makes much more sense and I misunderstood, updating.

Reply:
done!